specification.md

```diff
@@ -0,0 +1,148 @@
+# Specification
+
+The following values are part of Security Insights Specification `v1.0.0`
+
+## Header
+Here is an example of the header section, which holds high-level information about the project and its security documentation.
+
+```yaml
+header:
+  schema-version: 1.0.0
+  last-updated: '2021-09-01'
+  last-reviewed: '2022-09-01'
+  expiration-date: '2023-08-31T10:10:09.000Z'
+  project-url: https://github.com/foo/bar
+  commit-hash: 4dbf78ebc006ee5f668c0a74876ef8d6db9485be
+  project-release: '1.2.0'
+  changelog: https://github.com/foo/changelog.md
+  license: https://git.foo/license
+  parent-security-yaml: https://blah.com/ossf-security.yaml
+```
+
+Now let’s break that down:
+- `schema-version` (Required): Provide the version of the specification that you are adhering to.
+  - **String**
+- `last-updated` (Required): Date of the most recent change to this file.
+  - **Date**
+- `last-reviewed` (Required): Date this file was last reviewed to confirm holistic accuracy.
+  - **Date**
```

Comment:
Who needs to review it?
(Is there some authorization for review rights?)

Reply:
Updated to include "by a project maintainer"